src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/GregtechMetaTileEntity_IndustrialFishingPond.java

```diff
@@ -294,8 +301,8 @@ public boolean checkForWater() {
 			for (int j = mOffsetZ_Lower + 1; j <= mOffsetZ_Upper - 1; ++j) {
 				for (int h = 0; h < 2; h++) {
 					Block tBlock = aBaseMetaTileEntity.getBlockOffset(xDir + i, h, zDir + j);
-					// byte tMeta = aBaseMetaTileEntity.getMetaIDOffset(xDir + i, h, zDir + j);
-					if (tBlock == Blocks.air || tBlock == Blocks.flowing_water || tBlock == BlocksItems.getFluidBlock(InternalName.fluidDistilledWater)) {
+					byte tMeta = aBaseMetaTileEntity.getMetaIDOffset(xDir + i, h, zDir + j);
+					if (isNotStaticWater(tBlock, tMeta)) {
```

Comment:
do ore washer and algae farmer's water check need this fix?

Reply:
I think yes, but this way of fix didn't work for distilled water in washer.
I didn't know about Algae, let me see

Reply:
fixed for Algae